### DISCUSSION TITLE
Would the detectors work on a probe balloon?

### DISCUSSION THREAD

**Discussion by Lerryy** [ozel/DIY_particle_detector#47] [General]
Hi, i'm currently working on experiments for a suborbital flight (up to nearly 30 km, hopefully) and after founding your articles about electrons and alpha-particles i was wondering if the sensitive area of any of the diodes you used may be able to detect anything. That's because of the recommendations about placing the source of radiation as near as possible to the diodes. Also, how may the case be modified to be able to detect alpha-particles but still protect the diode from the noise caused by surrounding systems?

**Comment by ozel** [comment]
of course, you would detect much more radiation from the cosmic particle shower generated in the upper atmosphere - in particular, if the flight lasts long and is not just a very short parabola.
I would not bother about alphas and exposed diodes surfaces, but for general advice on blocking light in this case, have a look in the wiki https://github.com/ozel/DIY_particle_detector/wiki/Diodes.
I would stick with the electron detector design using 4 diodes behind a thin-walled but sturdy metal case. Strong ionising radiation like an occasional, stronger ionizing proton or pion and even some muons will leave a trace in the detector.
You may add 1 or 2 more diodes in parallel to the others, increasing the surface area. Keep in mind with more diodes, the noise of the amplifier circuit will increase and starts hiding smaller signal pulses (like from muons which are minimum ionizing = low energy deposit along its path through the diode).
